web/src/features/panels/zone/ZoneHeaderTitle.tsx

```diff
@@ -76,9 +77,11 @@ export default function ZoneHeaderTitle({ zoneId }: ZoneHeaderTitleProps) {
         </div>
         <TimeDisplay className="whitespace-nowrap text-sm" />
       </div>
-      <div className="ml-auto self-center px-3">
-        <ShareButton />
-      </div>
+      {useFeatureFlag('share-button') && (
```

Comment:
Optional improvement would be to make a variable `isShareButtonEnabled`, I think there's an argument to be made for either way of doing this but this variable approach fits with the existing style of how it's been done in the codebase previously

Reply:
Fair enough, consistency is good. Will update